verif/core_rrv/tests/bubblesort_lfsr.c

```diff
@@ -0,0 +1,89 @@
+
+//------------------------------------------------------------
+// Title : alive_lfsr
+// Project : core_rrv
+//------------------------------------------------------------
+// Description :
+//generating LENGTH pseudo random numbers created using LFSR
+//------------------------------------------------------------
+#include "interrupt_handler.h"
+
+#define LENGTH 15
+
+// Function to generate the next LFSR value
+unsigned int lfsr(unsigned int current_state) {
```

Comment:
Instead of writing again all the SW lfsr code, maybe add it to one of the .h file, and you can include the function